src/libraries/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.SquMul.cs

```diff
@@ -510,7 +493,7 @@ static void RightSmall(ReadOnlySpan<uint> left, ReadOnlySpan<uint> right, Span<u
                 Span<uint> bitsHigh = bits.Slice(n);
 
                 // ... compute low
-                MultiplyImpl(leftLow, right, bitsLow);
+                Multiply(leftLow, right, bitsLow);
```

Comment:
... the only downside (slight) is that this means there is now _one more_ test in this call path, which will provably always be false... so hopefully the JIT can reason that one out of existence.